quickstart/201-machine-learning-moderately-secure/network.tf

```diff
@@ -6,11 +6,27 @@ resource "azurerm_virtual_network" "default" {
   resource_group_name = azurerm_resource_group.default.name
 }
 
-resource "azurerm_subnet" "mlsubnet" {
-  name                 = "mlsubnet"
+resource "azurerm_subnet" "training-subnet" {
```

Comment:
We should use naming conventions per https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations to get it through the Terraform review.

Reply:
snet-training
snet-ml
etc.

Reply:
added proper naming conventions to resources

Reply:
thank!